common/coat_of_arms/template_lists/color_lists.txt

```diff
@@ -255,6 +255,86 @@ color_lists = {
 			125 = "black"
 			250 = "black_empire_dark_red"
 		}
+		
+		###### Warcraft Culture Specific CoAs ######
+		
+		### Humans ###
+		
+		special_selection = { 
+			trigger = { azerothian_coa_trigger = yes }
+			15  = "green_tirassian"
+			20  = "cyan_silverhand"
+			20  = "purple"
+			30  = "white"
```

Comment:
`white` is `metal_colors`. You can't use the same color in both color lists. Otherwise, you will get CoA with white emblem and white background.
Also, there's a rule to use brighter colors in `metal_colors` and darker colors in `normal_colors`.